features/step_definitions/common_steps.rb

```diff
@@ -114,9 +114,9 @@ def group_common(index)
   expect(hint_group.text).to have_content hint
 end
 
-Then(/^I am taken to step (\d+) ([^\"]*)$/) do |step, url|
+Then(/^I am taken to step (\d+) - ([^\"]*)$/) do |step, title|
   expect(common_page.content.p.text).to have_content step
-  expect(current_path).to eq "/questions/#{url}"
+  expect(common_page.content.h1.text).to eq title
```

Comment:
Using h1 doesnt seem right - we use page objects to abstract boring technical definitions like this - can the h1 not be called something meaningful from a user's perspective ?

Reply:
I know and I would appreciate your help on this but the user doesn't see the steps, only we do. I don't want to add ambiguity by calling the h1 something else